line_profiler/_logger.py

```diff
@@ -0,0 +1,286 @@
+        self._log(logging.WARNING, msg, *args, **kwargs)
+
+    def error(self, msg, *args, **kwargs):
+        self._log(logging.ERROR, msg, *args, **kwargs)
+
+    def critical(self, msg, *args, **kwargs):
+        self._log(logging.CRITICAL, msg, *args, **kwargs)
+
+    def configure(self, **kwargs):
+        """
+        For compatability with _StdlibLogBackend, all arguments are ignored.
+        The print backend does not need to be configured.
+        """
+        ...
+
+
+class _StdlibLogBackend(_LogBackend):
+    """
+    A wrapper for Python's standard logging.Logger.
+
+    The constructor optionally adds a StreamHandler (to stdout) and/or a logging.FileHandler if file is specified.
+
+    Example:
+        >>> import os
+        >>> import ubelt as ub
+        >>> import logging
+        >>> dpath = ub.Path.appdir('kwutil/test/logging').ensuredir()
+        >>> fpath = (dpath / 'test.log').delete()
+        >>> sl = _StdlibLogBackend('stdlib').configure(
+        >>>     level=logging.INFO,
+        >>>     stream={
+        >>>         'format': '%(asctime)s : [stream] %(levelname)s : %(message)s',
+        >>>     },
+        >>>     file={
+        >>>         'path': fpath,
+        >>>         'format': '%(asctime)s : [file] %(levelname)s : %(message)s',
+        >>>     }
+        >>> )
+        >>> sl.info('Hello %s', 'world')
+        >>> # Check that the log file has been written to
+        >>> text = fpath.read_text()
+        >>> print(text)
+        >>> assert text.strip().endswith('Hello world')
+    """
+
+    def __init__(self, name):
+        super().__init__(name)
+        self.logger = logging.getLogger(name)
+
+    def configure(
+        self,
+        level=None,
+        stream='auto',
+        file=None,
+    ):
+        """
+        Configure the underlying stdlib logger.
+
+        Parameters:
+            level: the logging level to set (e.g. logging.INFO)
+            stream: either a dict with configuration or a boolean/'auto'
+                - If dict, expected keys include 'format'
+                - If 'auto', the stream handler is enabled if no handlers are set
+                - If a boolean, True enables the stream handler.
+            file: either a dict with configuration or a path string.
+                - If dict, expected keys include 'path' and 'format'
+                - If a string, it is taken as the file path
+
+
+        Note:
+            For special attributes for the ``format`` argument of ``stream``
+            and ``file`` see
+            https://docs.python.org/3/library/logging.html#logrecord-attributes
+
+        Returns:
+            self (the configured _StdlibLogBackend instance)
+        """
+        if level is not None:
+            self.logger.setLevel(level)
+
+        # Default settings for file and stream handlers
+        fileinfo = {
+            'path': None,
+            'format': '%(asctime)s : [file] %(levelname)s : %(message)s'
```

Comment:
Should it be 
```suggestion
            'format': '%(asctime)s : [%(filename)s] %(levelname)s : %(message)s'
```
instead?

Or is the `[file]` just meant to be a plain, static prefix?